src/screens/OtherProfile/index.js

```diff
@@ -264,7 +185,7 @@ const OtherProfile = () => {
     };
     const result = await setUnFollow(data);
     if (result.code === 200) {
-      fetchOtherProfile();
+      refetchOtherProfile();
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Good use of the `refetchOtherProfile` function from the custom hook to refresh the profile after unfollowing a user.



<!-- This is an auto-generated comment by OSS CodeRabbit -->